go-controller/pkg/node/nodeController.go

```diff
@@ -0,0 +1,188 @@
+package node
+
+import (
+	"fmt"
+	cnitypes "github.com/containernetworking/cni/pkg/types"
+	ovncnitypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/cni/types"
+	"time"
+
+	nettypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+)
+
+type ovnNodeController struct {
+	name         string
+	Kube         kube.Interface
+	watchFactory factory.NodeWatchFactory
+
+	nadInfo    *util.NetAttachDefInfo
+	podHandler *factory.Handler
+	started    bool
+}
+
+func (n *OvnNode) initDefaultController() *ovnNodeController {
+	defaultNetConf := &ovncnitypes.NetConf{
+		NetConf: cnitypes.NetConf{
+			Name: types.DefaultNetworkName,
+		},
+		NetCidr:     config.Default.RawClusterSubnets,
```

Comment:
I used it in OVN package as this will make the code/logic the same among the default controller and secondary controller, (as the cluster subnet string is defined in NAD). Perhaps I should just fix doc? what is your suggestion?

Reply:
Let's wait for input from @jcaamano / @trozet ... I'm unsure this config is to be used or not. 

It might be OK to do as you say - just update the doc. Or we might ensure the `ClusterSubnets` are always getting the latest value. Not sure to be honest.